README.md

```diff
@@ -581,6 +583,66 @@ for other types of options.
 Many modifiers are polymorphic in this type argument, which means
 that they can be used with any builder.
 
+### Common Builder Patterns
+
+#### A list of values with a default
+
+We'll show 1 wrong ways to do it and then show 2 ways of implementing it.
+
+Wrong way: using `some (strOption modifiers)`/`many (strOption modifiers)`.
+We could use `some (strOption (long "arg-name" <> value "default"))`, which allows you to pass in values like this:
+`command --arg-name value1 --arg-name value2`
+
+However, combining `some`/`many` with a default `value` modifier guarantees that the parser will never terminate. Rather, it'll run forever and eventually your machine will run out of stack/memory and crash. Why? Because `some`/`many` work by parsing forever until they fail and then they return all the results they found. If the `value` modifier is added, then these parsers will never fail.
+
+Right way (but inconsistent): using `some (optionArgs) <|> NonEmptyString.singleton "default"`
```

Comment:
```diff
-<|>       NonEmptyString.singleton "default"
+<|> pure (NonEmptyList.singleton "default")
```

To avoid need for NonEmptyList we can use `many` as it results in `List` and do:
`many (optionArgs) <|> pure ("default": Nil)`, as it's more likely someone has seen `(:)/Cons` and `Nil` vs NonEmptyList

Reply:
Good point!

Reply:
Latest commits now include these points.